### ISSUE TITLE
Users with 0 Created (Teacher) Classrooms cannot access Teacher Classrooms

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/wildcam-gorongosa-education#185]
Issue: If a logged-in user with no created Teacher classrooms accesses https://learn.wildcamgorongosa.org/teachers/classrooms, the app crashes with an infinite spinner.

Analysis:
Error encountered is...

```
action @ 15:59:34.830 RECEIVE_CLASSROOMS 
main-c842746….min.js:34  prev state Object {teacher: Object, student: Object, login: Object, users: Object}
main-c842746….min.js:34  action Object {type: "RECEIVE_CLASSROOMS", data: Array[0], error: false, members: undefined}
main-c842746….min.js:34  error TypeError: Cannot read property 'map' of undefined(…)
main-c842746….min.js:34  next state Object {teacher: Object, student: Object, login: Object, users: Object}
main-c842746….min.js:34  action @ 15:59:34.836 RECEIVE_CLASSROOMS 
main-c842746….min.js:34  prev state Object {teacher: Object, student: Object, login: Object, users: Object}
main-c842746….min.js:34  action Object {type: "RECEIVE_CLASSROOMS", data: Array[0], error: true}
main-c842746….min.js:34  error TypeError: Cannot read property 'map' of undefined(…)
main-c842746….min.js:34  next state Object {teacher: Object, student: Object, login: Object, users: Object}
main-c8427463beb8d32a6a27.min.js:27 Uncaught (in promise) TypeError: Cannot read property 'map' of undefined(…)o @ main-c842746….min.js:27(anonymous function) @ main-c842746….min.js:34u @ main-c842746….min.js:24(anonymous function) @ main-c842746….min.js:34(anonymous function) @ main-c842746….min.js:34c.dispatch @ main-c842746….min.js:34(anonymous function) @ main-c842746….min.js:22
```

TL;DR of that mess of error messages: the `RECEIVE_CLASSROOMS` action/reducer encounters an undefined when there should be something.

Actions: Investigating.

**Comment by shaunanoordin** [comment]
Encountered: Infinite spinners
<img width="1280" alt="screen shot 2016-04-20 at 15 53 57" src="https://cloud.githubusercontent.com/assets/13952701/14679364/7352af94-0711-11e6-8497-649498c2e323.png">

Expected: A message saying "Sorry, no classrooms for you"
<img width="1280" alt="screen shot 2016-04-20 at 15 54 40" src="https://cloud.githubusercontent.com/assets/13952701/14679372/7c1adb56-0711-11e6-9fe8-89c2a3693e71.png">
